### PR TITLE
chore(docs): update content-watcher docs

### PR DESCRIPTION
**Pull request by enddynayn**
Update Content-Watcher-Service Documentation structure
and content.

### CODE REVIEW

**Review by saraswatpuneet — APPROVED** [review]
Much cleaner 🧹

File: developer-docs/content-watcher/README.md [hunk 1]
```diff
@@ -1,356 +1,186 @@
...
-- **Send content to registered webhooks**
-- **Simple Schema and MSA Id based filtering**
-
-#### Scanner API
-
-- **Start, Stop, and restart scanning**
+## 📖 About the Project <a name="about-project"></a>
 
-<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+The Content Watcher Service is part of the Gateway suite that provides a Web2-friendly interface for monitoring and retrieving blockchain content. It implements a webhook system that pushes updates to registered endpoints, making it easy to stay synchronized with chain events.
```
Comment (claireolmstead) [inline developer-docs/content-watcher/README.md:19]: Do we want to link to gateway?

File: developer-docs/content-watcher/README.md [hunk 2]
```diff
@@ -1,356 +1,186 @@
...
 
-<!-- LIVE Docs -->
+## 🔍 Architecture Overview <a name="architecture-overview"></a>
 
-## 🚀 Live Docs
+The Content Watcher Service provides a webhook-based system for receiving notifications about new content. Even with a crawl request, results are pushed to the webhook rather than retrieved directly.
```
Comment (claireolmstead) [inline developer-docs/content-watcher/README.md:23]: The second sentence is a little confusing to me. Thinking either we drop it or explain why we cannot retrieve it directly.

**Review by claireolmstead — COMMENTED** [review]
I think the big thing that this is missing is an explanation as to why we have to watch content, which should be near the top. We have to watch becuase things are happening async due to the nature of the chain and how we're queueing posts and puts on chain. I would like to see a little more of a technical description around that.

**Comment by enddynayn** [comment]
> I think the big thing that this is missing is an explanation as to why we have to watch content, which should be near the top. We have to watch becuase things are happening async due to the nature of the chain and how we're queueing posts and puts on chain. I would like to see a little more of a technical description around that.

Good point. Thanks! I update the documentation to reflect your suggestion.

**Review by claireolmstead — APPROVED** [review]
Great changes!

**Pull request merged**